mmtrack/datasets/got10k_dataset.py

```diff
@@ -44,18 +44,19 @@ def load_data_infos(self, split='train'):
         start_time = time.time()
         assert split in ['train', 'val', 'test', 'val_vot', 'train_vot']
         data_infos = []
-        with open(self.ann_file, 'r') as f:
-            # the first line of annotation file is dataset comment.
-            for line in f.readlines()[1:]:
-                # compatible with different OS.
-                line = line.strip().replace('/', os.sep).split(',')
-                data_info = dict(
-                    video_path=line[0],
-                    ann_path=line[1],
-                    start_frame_id=int(line[2]),
-                    end_frame_id=int(line[3]),
-                    framename_template='%08d.jpg')
-                data_infos.append(data_info)
+        data_infos_str = self.file_client.get_text(
+            self.ann_file).strip().split('\n')
```

Comment:
Can we use the `self.loadtxt()` API?

Reply:
I don't use `self.loadtxt()` API for the efficiency.
`self.loadtxt()` is fit for the load of the whole txt files in the NumPy format.

Reply:
I have readjusted it.